From 57ac2b8e060b88f9cb8c7c7c66ba094d5ca1e2a2 Mon Sep 17 00:00:00 2001
From: Nicolas Dufresne <nicolas.dufresne@collabora.com>
Date: Wed, 18 Dec 2024 13:35:53 -0500
Subject: [PATCH 097/117] va: display: Optimize out some property indirection

Because it was visible during some profiling, I thought it cost nothing
to optimize out the uneeded property get roundtrip.

Part-of: <https://gitlab.freedesktop.org/gstreamer/gstreamer/-/merge_requests/8179>
---
 gst-libs/gst/va/gstvadisplay.c | 6 +++---
 1 file changed, 3 insertions(+), 3 deletions(-)

diff --git a/gst-libs/gst/va/gstvadisplay.c b/gst-libs/gst/va/gstvadisplay.c
index e3f2f1174..1a32e3b9c 100644
--- a/gst-libs/gst/va/gstvadisplay.c
+++ b/gst-libs/gst/va/gstvadisplay.c
@@ -425,12 +425,12 @@ gst_va_display_initialize (GstVaDisplay * self)
 gpointer
 gst_va_display_get_va_dpy (GstVaDisplay * self)
 {
-  VADisplay dpy;
+  GstVaDisplayPrivate *priv;
 
   g_return_val_if_fail (GST_IS_VA_DISPLAY (self), NULL);
+  priv = GET_PRIV (self);
 
-  g_object_get (self, "va-display", &dpy, NULL);
-  return dpy;
+  return priv->display;
 }
 
 /**
-- 
2.43.0

